kava-stats.js

```diff
@@ -874,12 +897,18 @@ const updateDisplayValues = async (denoms) => {
   const marketData = await mapMarketData(denoms, markets)
   siteData['marketData'] = marketData;
 
-  const usdxMarketData = await mapUsdxMarketData(usdxMarketDataJson)
-  siteData['marketData']['usdx']['priceChangePercent'] = usdxMarketData
+  const usdxMarketData = await mapCoinGeckoApiData(usdxMarketDataJson)
+  siteData['marketData']['usdx']['priceChangePercent'] = usdxMarketData;
+
+  const swpMarketData = await mapCoinGeckoApiData(swpMarketDataJson)
```

Comment:
Implementing that renamed function to get price change data for SWP from CG's API